1.9/usage/cli/command-reference/dcos-task/dcos-task-exec.md

```diff
@@ -0,0 +1,34 @@
+---
+post_title: dcos task exec
+menu_order: 0
+---
+
+# Description
+Launch a process inside of a task container.
+
+# Usage
+
+```bash
+dcos task exec [OPTION]
+```
```

Comment:
Even if the parameters aren't optional? The <task> and <cmd> arguments are required, for example.